### DISCUSSION TITLE
Android 5.1

### DISCUSSION THREAD

**Discussion by coyoteclan** [termux/termux-app#3776] [General]
I wanted to ask if there is a version that would run on Android 5.1

**Comment by twaik** [comment]
Support for both app and packages was dropped for Android `5` and `6` on [2020-01-01](https://www.reddit.com/r/termux/comments/dnzdbs/end_of_android56_support_on_20200101/) at `v0.83`, however it was re-added just for the app *without any support for package updates* on [2022-05-24](https://github.com/termux/termux-app/pull/2740) via the [GitHub](#github) sources. Check [here](https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6) for the details.

**Comment by agnostic-apollo** [comment]
https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6
